solidity/dashboard/src/components/AmountForm.jsx

```diff
@@ -57,20 +34,29 @@ const AmountForm = ({
           normalize={normalizeAmount}
           format={formatFormAmount}
         />
-        <div className="text-caption text-grey-50">
-          {KEEP.displayAmountWithSymbol(availableAmount)} available.
+        <div className="flex row center" style={styles.availableAmountWrapper}>
```

Comment:
I think we should add `align-items: baseline` here becase the word "available" looks off kompared to the KEEP amount